packages/common-components/src/MenuDropdown/MenuDropdown.tsx

```diff
@@ -222,11 +227,13 @@ const MenuDropdown = ({
             {title}
           </Typography>
         )}
-        <Icon
+        {children && children}
```

Comment:
see the lgtm alert above.
```suggestion
        {children}
```